TicketBookingBackend/TicketBookingAPI/TicketBookingAPI/Program.cs

```diff
@@ -0,0 +1,38 @@
+using Microsoft.EntityFrameworkCore;
+using TicketBookingAPI.Repository;
+using TicketBookingAPI.Services;
+using TicketBookingData;
+
+var builder = WebApplication.CreateBuilder(args);
+
+// Add services to the container.
+
+builder.Services.AddControllers();
+// Learn more about configuring Swagger/OpenAPI at https://aka.ms/aspnetcore/swashbuckle
+builder.Services.AddEndpointsApiExplorer();
+builder.Services.AddSwaggerGen();
+
+builder.Services.AddAutoMapper(AppDomain.CurrentDomain.GetAssemblies());
+builder.Services.AddDbContext<PubContext>(
+    opt => opt.UseSqlServer(builder.Configuration.GetConnectionString("APIConnection"))
+    .UseQueryTrackingBehavior(QueryTrackingBehavior.NoTracking));
+
+builder.Services.AddScoped<CityDetailsRepository>();
+builder.Services.AddScoped<CityDetailsService>();
```

Comment:
Create an extension method to create a service and repository dependencies in one place